// Webezy.io Generated proto DO NOT EDIT
syntax = "proto3";


package webezy.OtherPackage.v1;



// [webezyio] - None
message OtherMessage {
	string StringField = 1;
}

